samples/nextjs/package.json

```diff
@@ -88,6 +89,7 @@
     "yaml-loader": "^0.6.0"
   },
   "scripts": {
+    "prepare": "cd ./node_modules/@sitecore-jss/sitecore-jss-nextjs && npm link ../../react",
```

Comment:
Called by lerna, when it bootstrap packages/samples. It solves problem of multiple instances of `react` when use `hooks`

Reply:
https://github.com/lerna/lerna/tree/main/commands/bootstrap